scripts/test-codex-runtime-noexec-staging.py

```
@@ -0,0 +1,115 @@
+    raise AssertionError(f"{path} is not an explicit mount")
+
+
+def main() -> int:
+    if os.geteuid() != 0:
+        raise AssertionError("the noexec regression must run as root")
+    options = mount_options(Path("/tmp"))
+    for required in ("rw", "noexec", "nosuid", "nodev"):
+        if required not in options:
+            raise AssertionError(f"/tmp is missing mount option {required}: {options}")
+
+    control = Path(f"/tmp/remote-dev-noexec-control-{os.getpid()}")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/remote-dev-noexec-control-"

(S108)